app/assets/javascripts/wizard/addon/components/wizard-preview-base.js

```diff
@@ -54,21 +55,74 @@ export default Component.extend({
 
   didInsertElement() {
     this._super(...arguments);
+    this.fontMap = PreloadStore.get("fontMap");
+    this.loadedFonts = new Set();
     const c = this.element.querySelector("canvas");
     this.ctx = c.getContext("2d");
     this.ctx.scale(scale, scale);
     this.reload();
   },
 
-  @observes(
-    "step.fieldsById.{color_scheme,body_font,heading_font,homepage_style}.value"
-  )
+  @observes("step.fieldsById.{color_scheme,homepage_style}.value")
   themeChanged() {
     this.triggerRepaint();
   },
 
+  @observes("step.fieldsById.{body_font}.value")
```

Comment:
I'm accepting new observers here for now 😛 we'll _glimmy_ that component at some point 😛

Reply:
Yeah I didn't like it either but didn't want to refactor the whole thing heh